src/main/java/org/mule/extensions/java/internal/transformer/ParameterTransformer.java

```diff
@@ -176,10 +178,11 @@ private Object transformParameter(Object value, ResolvableType parameterResolvab
       if (wrappedParameterType.isAssignableFrom(expressionValue.getClass())) {
         return expressionValue;
       }
-    } catch (ExpressionExecutionException e) {
+    } catch (ExpressionExecutionException | ExpressionRuntimeException e) {
       if (LOGGER.isDebugEnabled()) {
         LOGGER.debug(DATAWEAVE_TRANSFORMATION_ERROR_MESSAGE, value.getClass(), wrappedParameterType);
       }
+      return null;
```

Comment:
Are we ok returning null? what if the value was flagged as needed to be transformed but it would turn out to be valid for the java method?
Maybe only do this for the next minor?